### DISCUSSION TITLE
equivalence to CFF explorer

### DISCUSSION THREAD

**Discussion by ghost** [hasherezade/pe-bear#26] [Q&A] [answered]
hello, I currently use CFF explorer whenever I need to edit dlls and such, does this program work the same way as CFF? or rather, is it capable of doing the same things?

**Comment by hasherezade** [comment, marked as answer]
hello @cyberhardt !
yes, PE-bear serves the same purpose as CFF explorer, so it can be used as a replacement. However, it is not a clone of CFF explorer, (just yet another PE editor) so some features, and the way in which they are organized, will differ.
